README.md

```diff
@@ -192,15 +194,6 @@ batched jacobians:
 >>> assert jacobian.shape == (64, 5, 5)
 ```
 
-`jacrev` can be composed with itself to produce hessians:
-```py
->>> def f(x):
->>>   return x.sin().sum()
->>>
->>> x = torch.randn(5)
->>> hessian = jacrev(jacrev(f))(x)
-```
```

Comment:
I like this example but it's currently broken, erroring that reshape isn't supported (through a fallback or as a prim). We can get it to work with just sum, but then the hessian is all 0s and that's not super interesting

Reply:
wait, reshape isn't supported? That seems like something we should fix :P I'll take a look.

Reply:
Oh wait this might have been an old version--I pulled main and it looks like it's working okay. Sorry about the false alarm